src/components/AuthContainer.js

```diff
@@ -13,17 +13,17 @@ import TermsOfService from 'src/pages/TermsOfService'
 
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
-  const { isSignedIn, registrationStatus, acceptedTos, profile } = useStore(authStore)
+  const { isSignedIn, registrationStatus, acceptedTos, acceptedSamTos, profile } = useStore(authStore)
```

Comment:
Perhaps, I'll see what I can do. There was a time where we needed to keep track of which ToS wasn't yet accepted, but I think you're right that we can compute the ToS state and just store that value rather than two separate values.